### ISSUE TITLE
Raise error in Orbit.from_sbdb when more than one orbit is returned

### ISSUE THREAD

**Issue by shreyasbapat** [poliastro/poliastro#609]
🐞 **Problem**

When `Orbit.from_sbdb` is called with names which are too common, like "Halley" or "*alley", it returns a keyerror. Raise a suitable warning in those cases.

Code to reproduce error: 
```
Orbit.from_sbdb("Halley")
```

🖥 **Please paste the output of following commands**

* `conda info -a` (only if you have conda)
* `conda list` (only if you have conda)
* `pip freeze`

<details>
  
```bash
# Paste your output here:

```
</details>

* `python -c "import poliastro.testing; poliastro.testing.test()"`

<details>

```bash
# Paste your output here:

```
</details>

🎯 **Goal**

Related to #534 
To make `from_sbdb` more user friendly.

💡 **Possible solutions**

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

**Comment by Juanlu001** [comment]
Rather than a warning, I think we should raise an error, in the same way that we did with the old API:

![Screenshot_2019-03-18 Using NEOS package — poliastro 0 12 0 documentation](https://user-images.githubusercontent.com/316517/54541408-2b320380-499a-11e9-8898-b3adbfb79278.png)

**Comment by Ivan9528** [comment]
What about raising an error a bit more elaborated/understandable for the user about the different bodies found and then giving the user the chance to chose one of them by entering the complete name or the code number associated with it?

**Comment by Juanlu001** [comment]
@Ivan9528 thanks for chiming in! The problem with that is that it requires interactivity, and in a script that runs unattended it can block the program. #611 is unfinished, so perhaps you want to take care of this one?

**Comment by michiboo** [comment]
Hi!  is this issue still unsolved? if not I will try tackle it!

**Comment by Ivan9528** [comment]
Hi @michiboo!! It is already solved actually, I just have to make the pull request but the coding is done, I will do it tonight if time allows me to :D Thanks!!

**Comment by Ivan9528** [comment]
@michiboo I am sorry, I am not working on this issue but on #611. Anyway, @Juanlu001, please confirm that this issue is unsolved because I have run the code I have got a value error instead of a key one which I presume it was the purpose of the issue...

**Comment by shreyasbapat** [comment]
There was an effort to tackle this in #611 . But the PR author hasn't replied since. 

> I have run the code I have got a value error instead of a key one which I presume it was the purpose of the issue...

Did you run it after making changes for #611 ? Because #611 is solution for this issue :)

**Comment by Ivan9528** [comment]
Yes, I run it and it raises the error as the previous coder already did but changing slightly the coding. I am going to try to make the pull request so that you any can review it :D

**Comment by jorgepiloto** [comment]
Great @Ivan9528! :smile:

**Comment by Juanlu001** [comment]
Closed in #651.
